R/date_standardization_helpers.R

```diff
@@ -138,7 +149,10 @@ date_convert <- function(data, cols, error_tolerance,
     data[[cols]] <- new_dates
   }
 
-  return(data)
+  return(list(
+    data = data,
+    has_ambiguous_values = has_ambiguous_values
+  ))
```

Comment:
Please also update the documentation accordingly. And maybe NEWS as well?

Reply:
This is now documented.